src/screens/TopicMemberScreen/index.js

```diff
@@ -134,69 +149,6 @@ const TopicMemberScreen = () => {
     ShareUtils.shareCommunity(topicName);
   };
 
-  const showAnimationHeader = () => {
-    interactionManagerRef.current = InteractionManager.runAfterInteractions(() => {
-      Animated.timing(animatedHeight, {
-        toValue:
-          (coverPath
-            ? dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER
-            : dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT) +
-          dimen.size.TOPIC_FEED_HEADER_HEIGHT +
-          topPosition,
-        duration: 100,
-        useNativeDriver: false
-      }).start();
-      Animated.timing(opacityAnimationHeader, {
-        toValue: 1,
-        duration: 100,
-        useNativeDriver: false
-      }).start();
-    });
-    setHeaderHide(false);
-  };
-
-  React.useEffect(() => {
-    showAnimationHeader();
-  }, [coverPath]);
-
-  React.useEffect(() => {
-    const unsubscribe = navigation.addListener('focus', () => {
-      showAnimationHeader();
-    });
-
-    return () => {
-      if (interactionManagerRef.current) interactionManagerRef.current.cancel();
-      if (interactionManagerAnimatedRef.current) interactionManagerAnimatedRef.current.cancel();
-
-      unsubscribe();
-    };
-  }, [navigation]);
-
-  const handleScrollEvent = React.useCallback(
-    (event) => {
-      Keyboard.dismiss();
-      const {y} = event.nativeEvent.contentOffset;
-      if (y <= 15) {
-        showAnimationHeader();
-      } else {
-        interactionManagerAnimatedRef.current = InteractionManager.runAfterInteractions(() => {
-          Animated.timing(animatedHeight, {
-            toValue: dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT2 + topPosition,
-            duration: 100,
-            useNativeDriver: false
-          }).start();
-          Animated.timing(opacityAnimationHeader, {
-            toValue: 0,
-            duration: 100,
-            useNativeDriver: false
-          }).start();
-        });
-        setHeaderHide(true);
-      }
-    },
-    [coverPath, animatedHeight]
-  );
-
   const onTokenCancel = () => {
     cancelTokenRef?.current?.cancel();
     cancelTokenRef.current = axios.CancelToken.source();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onTokenCancel` function cancels any ongoing axios request when called. This is a good practice to prevent memory leaks from unresolved promises.
```javascript
const onTokenCancel = () => {
  cancelTokenRef?.current?.cancel();
  cancelTokenRef.current = axios.CancelToken.source();
};
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->